benchmark_db_terraform/README.md

```diff
@@ -0,0 +1,66 @@
+# Overview
+
+This folder contains an executable description of the Forest service, which has been developed to assist in testing Forest database backends and their configurations. Additionally, the script also offers the capability to perform benchmarking, specifically measuring the "daily" benchmarks for Forest and Lotus snapshot import times (in seconds) and validation times (in tipsets per second).
+The benchmark results are subsequently uploaded to an S3 bucket, where they are stored in both a weekly file and an all_result file.
```

Comment:
We're not testing database configurations. The benchmarking suite grew out of a tool that benchmarked database configurations, but we're benchmarking something else with this service.